tests/unit/test_api_diagnostics.py

```
@@ -1383,3 +1383,268 @@ def test_ciissversion_validated_and_declared_keys_present(self):
+        incomplete_relays = [
+            {
+                **base, 'fingerprint': fp,
+                'effective_family': ['A' * 40],
+                'aroi_domain': 'none', 'aroi_configured': False,
+                'contact': 'url:1aeo.com proof:uri-familyid-ed25519',
+                'contact_md5': f'sib_{i}',
+            }
+            for i, fp in enumerate(sibling_fps)
+        ]
+        rs.json = {
+            'relays': [validated] + incomplete_relays,
```

Ruff (0.15.12)

[warning] Consider `[validated, *incomplete_relays]` instead of concatenation

Replace with `[validated, *incomplete_relays]`

(RUF005)